src/SoCommunicator.php

```diff
@@ -324,4 +324,33 @@ private function StripHash($suffixed)
 
         return $remittanceIdentifier;
     }
+
+
+    /**
+     * Executes a refund request using the EpsRefundRequest object.
+     *
+     * @param EpsRefundRequest $refundRequest The refund request data to be sent.
+     * @param string|null $targetUrl Optional endpoint URL for EPS refund requests. Defaults to the base URL with refund path.
+     * @param string|null $logMessage Optional custom log message for tracking the refund process.
+     * @return string The response body from the refund request.
+     * @throws HttpResponseException If the request fails with a non-200 HTTP status code.
+     */
+    public function ProcessRefund(EpsRefundRequest $refundRequest, $targetUrl = null, $logMessage = null)
+    {
+        $this->WriteLog($logMessage ?? 'Initiating refund request.');
+
+        if ($targetUrl === null)
+            $targetUrl = $this->BaseUrl . '/refund/eps/v2_6';
```

Comment:
Can you tell me where you got the URL path from? I can't find this in the EPS documentation.